src/xdocs/config_imports.xml

```diff
@@ -1426,6 +1426,16 @@ import java.util.stream.IntStream;
             <td>false</td>
             <td>3.2</td>
           </tr>
+          <tr>
+          <td>separatedStaticGroups</td>
+              <td>
+              whether static import groups should be separated by, at least, one
+              blank line or comment and aren't separated internally.
+              </td>
```

Comment:
I think we should mention in xdoc that new property is efficient only when `option` is "top" and "bottom".

The same is better to do for `staticGroups`.

Reply:
done.